.circleci/config.yml

```diff
@@ -0,0 +1,65 @@
+version: 2.1
```

Comment:
Let me know if we should use another version of this config.yml file @mieubrisse I wasn't sure if we should publish the artifact because this repository will be only cloned and not downloaded for Docker Hub, also I removed the run application part.